project/Dependencies.scala

```diff
@@ -40,7 +40,7 @@ object Dependencies {
   private val ficusV = "1.5.1"
   // The "com.vladsch.flexmark" % "flexmark-profile-pegdown" % flexmarkV dependency is an implicit, version-specific
   // runtime dependency of ScalaTest. At the time of this writing this is the newest version known to work.
```

Comment:
I wish I remember what "known to work" meant, that's a pretty big jump in the version of this dependency for a patch release bump in ScalaTest. Anyhoo I think this comment may be obsolete now.

Reply:
Scalatest requires flexmark, but for some reason doesn't bring it in itself (I double-checked this removing flexmark from Dependencies, which bombed). But the version of flexmark needs to match what Scalatest expects. For the release of Scalatest that we are upgrading to, they want version 0.62.2 (see notes in https://github.com/scalatest/scalatest/releases/tag/release-3.2.10). Upgrading Scalatest without upgrading flexmark failed.

I think the comment is accurate, though a bit obtuse.

Reply:
Perhaps add some of this detail to the `scalatestV` comment below? It's kind of a weird situation that Scalatest doesn't express its actual dependency on flexmark...